Common/src/main/java/com/tabnineCommon/lifecycle/LifecycleInitializer.kt

```diff
@@ -0,0 +1,12 @@
+package com.tabnineCommon.lifecycle
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.components.ServiceManager
+import com.intellij.openapi.fileEditor.FileEditorManagerListener
+
+fun initializeLifecycleEndpoints() {
+    ServiceManager.getService(BinaryStateService::class.java).startUpdateLoop()
+    ApplicationManager.getApplication().messageBus.connect().subscribe(
```

Comment:
can this be part of the `TabnineFileEditorListener` ctor?  or at least some `registerFileEditorListener` function